docker-compose.dev.yml

```diff
@@ -150,3 +150,13 @@ services:
     restart: "no"
   berichtencentrum-melding:
     restart: "no"
+  lmb-ldes-consumer:
+    networks:
+      - default
+      - debug
+  lmb-upsert-ldes-entity:
+    image: lblod/ldes-make-object-service:latest
```

Comment:
can this be removed?

Reply:
yeah we should update the image when a new version has been released, but for now we can just use the image used in de docker-compose.yml